mobile/components/artist_app_banner/index.coffee

```diff
@@ -22,4 +22,8 @@ module.exports = ->
   $el.find('.artist-app-banner-close-button').on 'click', (e) ->
     e.preventDefault()
     $(loginClass).show()
-    banner.remove()
\ No newline at end of file
+    banner.remove()
+
+  $el.find('.artist-app-banner-button').on 'click', (e) ->
+    $img = $('<img src="https://app.adjust.com/r0hk13" />')
```

Comment:
Hah, are you just abusing the `img` tag to perform a GET or does it actually return like a spacer gif or something?